--- tests/www.geogebra.org/en/upload/files/Cares_alumnos/Cedron/Teorema de Simson.out	2011-07-23 23:31:44.234646827 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Cares_alumnos/Cedron/Teorema de Simson.out	2011-07-24 09:14:15.082364773 +0200
@@ -7,7 +7,7 @@
 e: -2x - 3y = -33.07046736
 f: 6.26x + 1.26y = 44.47383292
 g: -4.26x + 1.74y = -11.40336556
-h: 3x - 2y = 19
+h: -3x + 2y = -19
 G: (9.47237959, 4.70856939)
 E: (3.64501574, 2.37034568)
 F: (6.40448591, 3.47758023)
