--- tests/www.geogebra.org/en/upload/files/Cajgel2/Obsah trojuholn%EDka vstvorcovej sieti.out	2011-07-23 23:31:08.244646813 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Cajgel2/Obsah trojuholn%EDka vstvorcovej sieti.out	2011-07-24 09:13:01.349249794 +0200
@@ -1,13 +1,13 @@
 A: (0, 0)
-a: 10
-b: 3
+a: 7
+b: 9
 n-uholník1: 50
 a_1: 10
 c: 5
 b_1: 10
 d: 5
-e: 8
-f: 1
+e: 10
+f: 3
 F: (0, 0)
 G: (10, 0)
 n-uholník2: 20
