--- tests/www.geogebra.org/en/upload/files/Brasil/S%E9rgio Dantas/teorema_pitagoras.out	2011-07-23 23:28:40.384646759 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Brasil/S%E9rgio Dantas/teorema_pitagoras.out	2011-07-24 09:07:32.926489887 +0200
@@ -19,7 +19,7 @@
 p: x² + (y - 6)² = 61
 l: -5x + 6y = 36
 H: (6, 11)
-m: 6x + 5y = 91
+m: -6x - 5y = -91
 n: -5x + 6y = -25
 I: (11, 5)
 polígono1: 15
@@ -27,7 +27,7 @@
 o: 7.81024968
 b_1: 6
 q: x = 2.5
-J: undefined
+J: nem definiált
 K: (2.5, 0)
 L: (2.5, -5)
 r: 5
@@ -59,8 +59,8 @@
 n_2: 0
 d_2: 5
 f_1: x² + y² - 10x = -25
-V: (5, 0)
-g_1: 6x + 5y = 30
+V: (?, ?)
+g_1: -6x - 5y = -30
 W: (0, 6)
 polígono5: 0
 c_3: 0
@@ -73,8 +73,8 @@
 f_2: 6
 g_2: 0
 h_1: x² + y² - 10x = -25
-Z: (5, 0)
-i_1: 6x + 5y = 30
+Z: (?, ?)
+i_1: -6x - 5y = -30
 A_1: (0, 6)
 polígono7: 0
 a_2: 0
