--- tests/www.geogebra.org/en/upload/files/Brasil/Heitor Fragoso/Circulo_trigonometrico.out	2011-07-23 23:27:30.004646733 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Brasil/Heitor Fragoso/Circulo_trigonometrico.out	2011-07-24 09:04:59.386489931 +0200
@@ -17,11 +17,11 @@
 k: 1
 l: 0
 n: y = 1
-I: undefined
+I: nem definiált
 p: ?
 q: ?
 r: x = 1
-J: undefined
+J: nem definiált
 K: (1, 0)
 s: ?
 t: 1
