--- tests/www.geogebra.org/en/upload/files/BCCC/SVC/3 point moving average.out	2011-07-23 23:06:38.294646267 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/SVC/3 point moving average.out	2011-07-24 08:49:44.206490191 +0200
@@ -77,7 +77,7 @@
 k_2: 1
 l_2: 1
 m_2: y = 8.98
-B16: 1990
+B16: 2005
 n_2: x = 12
 p_2: x = 13
 q_2: x = 14
@@ -152,16 +152,16 @@
 s_4: 1
 t_4: 1
 a_5: 1
-B2: 136
-C2: 152
-D2: 70
-E2: 158
-B3: 228
-B4: 108
-C3: 140
-C4: 222
-D3: 109
-D4: 107
-E3: 244
-E4: 239
-C11: 1
+B2: 181
+C2: 236
+D2: 144
+E2: 141
+B3: 117
+B4: 170
+C3: 248
+C4: 274
+D3: 92
+D4: 126
+E3: 300
+E4: 104
+C11: 7
