--- tests/www.geogebra.org/en/upload/files/BCCC/SVC/4 point moving average.out	2011-07-23 23:06:40.734646267 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/SVC/4 point moving average.out	2011-07-24 08:49:48.556490190 +0200
@@ -77,7 +77,7 @@
 k_2: 1
 l_2: 1
 m_2: y = 8.98
-B16: 2007
+B16: 1996
 n_2: x = 12
 p_2: x = 13
 q_2: x = 14
@@ -140,19 +140,19 @@
 B_3: (13, 5)
 C_3: (14, 5)
 D_3: (15, 5)
-B2: 111
-C2: 255
-D2: 65
-E2: 215
-B3: 134
-B4: 105
-C3: 139
-C4: 144
-D3: 129
-D4: 144
-E3: 235
-E4: 115
-C11: 6
+B2: 104
+C2: 237
+D2: 145
+E2: 102
+B3: 235
+B4: 235
+C3: 203
+C4: 296
+D3: 87
+D4: 118
+E3: 165
+E4: 257
+C11: 4
 i_4: x = 5.5
 j_4: x = 6.5
 k_4: x = 7.5
