--- tests/www.geogebra.org/en/upload/files/BCCC/SVC/Missing angle.out	2011-07-23 23:17:50.934646518 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/SVC/Missing angle.out	2011-07-24 08:50:25.746490179 +0200
@@ -1,5 +1,5 @@
-B7: 19
-B8: 1
+B7: 8
+B8: 24
 D: (6.90329198, 2.38967592)
 α: 319.69026734°
 B13: 40.30973266°
@@ -13,5 +13,5 @@
 F: (-0.19416159, 0.97216134)
 i: 2.29080748
 γ: 139.69026734°
-m: 1
+m: 6
 a: 319.69026734°
