--- tests/www.geogebra.org/en/upload/files/BCCC/SVC/Stem and Leaf.out	2011-07-23 23:18:19.864646528 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/SVC/Stem and Leaf.out	2011-07-24 08:51:23.896490163 +0200
@@ -1,18 +1,18 @@
-A2: 9
-A16: 11
-A15: 17
-A14: 21
-A13: 21
-A12: 16
-A11: 29
-A10: 25
-A9: 13
-A8: 3
-A7: 17
-A6: 21
-A5: 11
-A4: 2
-A3: 30
+A2: 6
+A16: 29
+A15: 23
+A14: 26
+A13: 7
+A12: 11
+A11: 13
+A10: 28
+A9: 6
+A8: 15
+A7: 28
+A6: 16
+A5: 15
+A4: 24
+A3: 7
 list4: {6, 6, 8, 9, 11, 16, 17, 17, 20, 22, 23, 25, 25, 26, 27}
 D2: 8
 D3: 100
@@ -156,10 +156,10 @@
 M14: 
 M15: 
 M16: 
-text2: \begin{array}{lllllllllllllll}6&6&8&9& & & & & & & & & & &  \\ \end{array}
-text3: \begin{array}{lllllllllllllll}1&6&7&7& & & & & & & & & & &  \\ \end{array}
-text4: \begin{array}{lllllllllllllll}0&2&3&5&5&6&7& & & & & & & &  \\ \end{array}
-text1: \begin{array}{lllllllllllllll}8&23&6&25&17&27&9&26&6&25&17&16&22&20&11 \\ \end{array}
+text2: {\left.\begin{array}{lllllllllllllll}\text{6}&\text{6}&\text{8}&\text{9}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text3: {\left.\begin{array}{lllllllllllllll}\text{1}&\text{6}&\text{7}&\text{7}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text4: {\left.\begin{array}{lllllllllllllll}\text{0}&\text{2}&\text{3}&\text{5}&\text{5}&\text{6}&\text{7}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text1: {\left.\begin{array}{lllllllllllllll}8&23&6&25&17&27&9&26&6&25&17&16&22&20&11 \\ \end{array}\right.}
 c: 2.36008474
 d: 0.02x + 2.36y = 18.6736
 D: (0.06705063, 7.91197415)
