--- tests/www.geogebra.org/en/upload/files/AHS_Fouss/Joseph A-1.out	2011-07-23 22:57:07.614646054 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/AHS_Fouss/Joseph A-1.out	2011-07-24 08:30:13.646490524 +0200
@@ -1 +1 @@
-f: 0.00551984 x⁵ - 0.15733005 x⁴ + 1.70646756 x³ - 8.84277083 x² + 22.55053423 x - 20.86515061
+f: 0.00551984x⁵ - 0.15733005x⁴ + 1.70646756x³ - 8.84277083x² + 22.55053423x - 20.86515061
