--- tests/www.geogebra.org/en/upload/files/AHS_Fouss/Picture 003.jpg.out	2011-07-23 22:56:56.374646050 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/AHS_Fouss/Picture 003.jpg.out	2011-07-24 08:29:49.606490531 +0200
@@ -1 +1 @@
-f: -0.00021024 x⁶ + 0.00525933 x⁵ - 0.03343411 x⁴ - 0.04996119 x³ + 0.72144489 x² + 0.10863152 x + 0.482244
+f: -0.00021024 x⁶ + 0.00525933x⁵ - 0.03343411x⁴ - 0.04996119x³ + 0.72144489x² + 0.10863152x + 0.482244
