--- tests/www.geogebra.org/en/upload/files/COLOMBIA/Toribio Arrieta/Par%E1bola M%FAltiple.out	2011-07-23 23:29:16.504646773 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/COLOMBIA/Toribio Arrieta/Par%E1bola M%FAltiple.out	2011-07-24 09:08:59.840064862 +0200
@@ -5,7 +5,7 @@
 Directriz: 9.07546484x - 0.02703114y = 19.05597148
 Foco: (0, 0)
 d: 0.00073068x² + 0.49064038x y + 82.36406198y² + 345.88359812x - 116.9717221y = 322.32853075
-E: (0.93096113, -0.00277285)
+E: (0.75857695, -0.40071609)
 b: 0.02703114x + 9.07546484y = -30.8899022
 F: (2.08956769, -3.40989538)
 e: 5.06444768
