--- tests/www.geogebra.org/en/upload/files/Bulgarian/%D0%E5%F6%E8%EF%F0%EE%F7%ED%E8 %F3%F0%E0%E2%ED%E5%ED%E8%FF/reciprochni_uravneniya_worksheet.out	2011-07-23 23:28:50.964646763 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Bulgarian/%D0%E5%F6%E8%EF%F0%EE%F7%ED%E8 %F3%F0%E0%E2%ED%E5%ED%E8%FF/reciprochni_uravneniya_worksheet.out	2011-07-24 09:08:00.966489879 +0200
@@ -1,5 +1,5 @@
-A: (-1.79074314, 0)
+A: (-1.79074314, -0.0000002)
 B: (-1, 0)
-C: (-0.55842738, 0)
-D: (0.22430588, 0)
-E: (4.45819798, 0)
+C: (-0.55842741, -0.00000022)
+D: (0.22430585, 0.00000053)
+E: (4.45819798, 0.00000026)
