--- tests/www.geogebra.org/en/upload/files/BELARUS/Higher mathematics/Visualize_Integrals.out	2011-07-23 23:22:59.704646631 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BELARUS/Higher mathematics/Visualize_Integrals.out	2011-07-24 08:56:17.326490079 +0200
@@ -1,5 +1,5 @@
 f: x²
-h: -(x) sin(x) + 3
+h: -x sin(x) + 3
 g: x
 i: 5
 k: 18
@@ -10,15 +10,15 @@
 r: 18.25
 t: 7.5
 u: 23.9258075
-z: 22.74999814
-a_1: 13.75000126
+z: 22.74999908
+a_1: 13.75000054
 f_1: 8.54166627
-g_1: 6.45833371
-l_1: 26.8232156
-m_1: 21.03776862
+g_1: 6.45833333
+l_1: 26.82321582
+m_1: 21.03776779
 r_1: 25
 s_1: 25
 t_1: 25
-a_2: 21.83059319
-b_2: 27.16253037
+a_2: 21.82987461
+b_2: 27.16253062
 c_2: 24.49586418
