--- tests/www.geogebra.org/en/upload/files/AlgGeoC/Chapter 10/CirclesSegments.out	2011-07-23 23:00:04.124646120 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/AlgGeoC/Chapter 10/CirclesSegments.out	2011-07-24 08:36:57.186490409 +0200
@@ -32,7 +32,7 @@
 t: 5.35297766
 N: (-2.4648768, -2.35295632)
 M: (3.64430634, -0.96505887)
-distanceNt: 5.63391718
+distanceNt: 6.0089716
 distanceKN: 9.33456014
 distanceMK: 3.06970756
 c_1: (x - 2.12)² + (y + 1.22)² = 9.8228
