--- tests/www.geogebra.org/en/upload/files/BCCC/CLN/Rectangles with Algebra.out	2011-07-23 23:05:45.844646247 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/CLN/Rectangles with Algebra.out	2011-07-24 08:48:00.626490221 +0200
@@ -3,10 +3,10 @@
 b: 3.52
 c: 6.86
 d: 3.52
-C1: 16
-D1: 6
-text1: \begin{array}{lll}x&+&3 \\ \end{array}
-text2: \begin{array}{l}9 \\ \end{array}
-text4: \begin{array}{ll}Area=&207 \\ \end{array}
-text3: \begin{array}{ll}x=&20 \\ \end{array}
-text5: \begin{array}{l}9(\,x\,+3)\,=\,207 \\ 9\,x\,+27\,=\,207 \\ \end{array}
+C1: 20
+D1: 19
+text1: {\left.\begin{array}{lll}\text{x}&\text{+}&3 \\ \end{array}\right.}
+text2: {\left.\begin{array}{l}9 \\ \end{array}\right.}
+text4: {\left.\begin{array}{ll}\text{Area=}&207 \\ \end{array}\right.}
+text3: {\left.\begin{array}{ll}\text{x=}&20 \\ \end{array}\right.}
+text5: {\left.\begin{array}{l}\text{9(\,x\,+3)\,=\,207} \\ \text{9\,x\,+27\,=\,207} \\ \end{array}\right.}
