--- tests/www.geogebra.org/en/upload/files/BCCC/CLN/Simple Quadratic Tables.out	2011-07-23 23:06:02.884646253 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/CLN/Simple Quadratic Tables.out	2011-07-24 08:48:32.836490211 +0200
@@ -1,7 +1,7 @@
 d: 1
 f: 1
-m: -1
-c: 3
-text1: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y& & & & & & &  \\ \end{array}
-text3: y = x² + x + 2
-text2: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y&8&4&2&2&4&8&14 \\ \end{array}
+m: 1
+c: 0
+text1: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text3: y = x^{2} + x + 2
+text2: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&8&4&2&2&4&8&14 \\ \end{array}\right.}
