--- tests/www.geogebra.org/en/upload/files/BCCC/CLN/Surface Area Cuboid.out	2011-07-23 23:06:04.904646253 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/CLN/Surface Area Cuboid.out	2011-07-24 08:48:36.616490210 +0200
@@ -1,8 +1,8 @@
-B6: 7
+B6: 8
 a: 2.98964881
 b: 1.44x + 2.62y = -1.7712
 c: 1.44x + 2.62y = 15.82878949
-B4: 5
+B4: 3
 d: 2.62x - 1.44y = 15.46
 e: 2.62x - 1.44y = 36.5694944
 f: 3.01236877
@@ -37,6 +37,6 @@
 h_1: 5.88697556
 i_1: 7.06086091
 distanceJI: 7.06086091
-B5: 3
+B5: 8
 distanceNO: 5.88697556
 distanceJM: 8.45692335
