--- tests/www.geogebra.org/en/upload/files/BCCC/CLN/Table Straight Line.out	2011-07-23 23:06:09.174646255 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/BCCC/CLN/Table Straight Line.out	2011-07-24 08:48:44.196490208 +0200
@@ -1,7 +1,7 @@
-m: 1
-c: -2
-text1: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y& & & & & & &  \\ \end{array}
-text2: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y&7&4&1&-2&-5&-8&-11 \\ \end{array}
+m: -3
+c: 3
+text1: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text2: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&7&4&1&-2&-5&-8&-11 \\ \end{array}\right.}
 text3: y = -3x - 2
-text6: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y&7& & &-2& & &  \\ \end{array}
-text7: \begin{array}{llllllll}x&-3&-2&-1&0&1&2&3 \\ y&7& &1&-2& & &-11 \\ \end{array}
+text6: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&7&\text{\;}&\text{\;}&-2&\text{\;}&\text{\;}&\text{\;} \\ \end{array}\right.}
+text7: {\left.\begin{array}{llllllll}\text{x}&-3&-2&-1&0&1&2&3 \\ \text{y}&7&\text{\;}&1&-2&\text{\;}&\text{\;}&-11 \\ \end{array}\right.}
