--- tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions03.out	2011-07-23 23:03:27.844646195 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions03.out	2011-07-24 08:42:50.396490309 +0200
@@ -1,9 +1,9 @@
 h: x = 4
 A: (4, 0)
 g: x = 4
-c': 1
-b': -1
-n': 1
+c': -1
+b': -2
+n': 2
 v_a: (0, 1)
 Asints: (?, ?)
 asinver: ?x = ?
@@ -13,8 +13,8 @@
 u: (0, 1)
 v_{ln}: (0, 1)
 fsol: -2 (x + 1)³ - 4
-colorf: If[x ≤ 3, 1, If[x ≤ 6, 1, 0]]
-colorg: If[x ≤ 3, 0, If[x ≤ 6, 1, 1]]
+colorf: Ha[x ≤ 3, 1, Ha[x ≤ 6, 1, 0]]
+colorg: Ha[x ≤ 3, 0, Ha[x ≤ 6, 1, 1]]
 u': (0, 1)
 u'': (0, 1)
 u_{nalax}: (0, 2)
