--- tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions.out	2011-07-23 23:03:25.084646195 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions.out	2011-07-24 08:42:45.336490310 +0200
@@ -1,6 +1,6 @@
-c': -3
+c': -2
 b': -5
-n': 1
+n': 2
 v_a: (0, 1)
 Asints: (?, ?)
 asinver: ?x = ?
@@ -10,8 +10,8 @@
 u: (0, 1)
 v_{ln}: (0, 1)
 fsol: -2 (x + 1)² - 3
-colorf: If[x ≤ 3, 1, If[x ≤ 6, 1, 0]]
-colorg: If[x ≤ 3, 0, If[x ≤ 6, 1, 1]]
+colorf: Ha[x ≤ 3, 1, Ha[x ≤ 6, 1, 0]]
+colorg: Ha[x ≤ 3, 0, Ha[x ≤ 6, 1, 1]]
 u': (0, 1)
 u'': (0, 1)
 u_{nalax}: (0, 2)
