--- tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions02.out	2011-07-23 23:03:26.374646195 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Armando Landa/1ParametrFunctions02.out	2011-07-24 08:42:47.736490310 +0200
@@ -1,5 +1,5 @@
-c': 0
-b': -1
+c': -3
+b': -2
 n': 1
 v_a: (0, 1)
 Asints: (?, ?)
@@ -10,8 +10,8 @@
 u: (0, 1)
 v_{ln}: (0, 1)
 fsol: -(x + 4)¹ + 1
-colorf: If[x ≤ 3, 1, If[x ≤ 6, 1, 0]]
-colorg: If[x ≤ 3, 0, If[x ≤ 6, 1, 1]]
+colorf: Ha[x ≤ 3, 1, Ha[x ≤ 6, 1, 0]]
+colorg: Ha[x ≤ 3, 0, Ha[x ≤ 6, 1, 1]]
 u': (0, 1)
 u'': (0, 1)
 u_{nalax}: (0, 2)
