--- tests/www.geogebra.org/en/upload/files/Armando Landa/4QuadraticFunctionsApril2011.out	2011-07-23 23:03:39.054646199 +0200
+++ tmp/tests/www.geogebra.org/en/upload/files/Armando Landa/4QuadraticFunctionsApril2011.out	2011-07-24 08:43:19.736490301 +0200
@@ -8,8 +8,8 @@
 perpR'_{bis}: x = 4
 Vf2: (3, 1)
 Vf3: (1, 2)
-b': -2
-c': -3
+b': 1
+c': -2
 n': 2
 d: y = 1
 e: x = -1
@@ -35,8 +35,8 @@
 segf_2: 48
 segf'_3: 10
 g: 79
-colorf: If[x ≤ 1, 1, If[x ≤ 2, 1, 0]]
-colorg: If[x ≤ 1, 0, If[x ≤ 2, 1, 1]]
+colorf: Ha[x ≤ 1, 1, Ha[x ≤ 2, 1, 0]]
+colorg: Ha[x ≤ 1, 0, Ha[x ≤ 2, 1, 1]]
 n'': 2
 vecAf2: (0, 1)
 seghf_1: 1
